docs/source/ops/index.rst

```diff
@@ -0,0 +1,86 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. ops-ref_index:
+
+DAML Participant pruning
+========================
+
+.. HINT::
+   DAML Participant pruning is currently an :doc:`Early Access Feature in Labs status </support/status-definitions>`
+
+The DAML Ledger API exposes an append-only ledger model; on the other hand, DAML Participants must be able to operate continuously for an indefinite amount of time on a limited amount of hot storage.
+
+In addition, privacy demands [1]_ may require removing Personally Identifiable Information (PII) upon request.
+
+To satisfy these requirements, the :ref:`Pruning Service <com.daml.ledger.api.v1.admin.ParticipantPruningService>` Ledger API endpoint [2]_ allows DAML Participants to support pruning of DAML contracts and transactions that were respectively archived and submitted before or at a given ledger offset.
```

Comment:
Suggesting to make this more precise to match the specific pruning deletes here, i.e. we prune events and completions (archived contracts are already removed independently form pruning from `participant_contracts`), and we also prune `participant_command_completions`: https://github.com/digital-asset/daml/blob/f0b98a85b9beb06f5fe4619aaba7b0f6c7ae18f0/ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala#L865-L866

The situation with transactions is a bit more subtle when we refer to the `TransactionService.GetTransactionTrees` where we end up potentially returning "partial" transactions with only `CreatedEvent` on behalf of contracts that have not been archived as of the specified pruning offset. But I am not sure how to capture this in a concise way without this being super-confusing.

```suggestion
To satisfy these requirements, the :ref:`Pruning Service <com.daml.ledger.api.v1.admin.ParticipantPruningService>` Ledger API endpoint [2]_ allows DAML Participants to support pruning of events on behalf of DAML contracts and command completions that were respectively archived and completed before or at a given ledger offset.
```

Reply:
That's not correct though, right? We don't return "partial" transactions. We retain create events in the `participant_events` table (if the respective contract hasn't been archived yet) so that we can serve the ActiveContractsService. But we don't don't deliver these create events as part of the transaction services, because the minimum starting offset of transaction streams must be greater than the last pruned offset.

Reply:
You're right - I forgot that we end up hiding this part. Good. So this speaks to the second paragraph in my comment above - will try to cross that out as the rest is still relevant I think.

Reply:
@oliverse-da I applied your suggestions about this. Thanks!